resources/epilepsy/ResourceFile_Epilepsy/additional info.csv

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
+oid sha256:23f3b4cef6927114b414344c95a4c262691312049ce7dc101042230e895ff066
```

Comment:
do we need to keep this file?

Reply:
@andrew-phillips-1 , should I delete this file?